### ISSUE TITLE
Percentages are not labeled 🏷

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#270]
'Tracker Networks Top Offender' is not labeled. It just says 29%, 6%, etc. What does that percentage signify?

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43592864-7983852c-9644-11e8-9703-99d29f12cd8e.png">
